docs/docs/cmd/flow/run/run-get.md

```diff
@@ -97,17 +107,106 @@ m365 flow run get --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5
 === "Text"
 
     ```text
-    endTime    :
-    name       : 08585329112602833828909892130CU17
-    startTime  : 2022-11-17T14:33:45.2763872Z
-    status     : Running
-    triggerName: When_a_new_response_is_submitted
+    endTime    : 2023-03-04T09:05:22.5880202Z
+    name       : 08585236861638480597867166179CU104
+    startTime  : 2023-03-04T09:05:21.8066368Z
+    status     : Succeeded
+    triggerName: When_an_email_is_flagged_(V4)
     ```
 
 === "CSV"
 
     ```csv
     name,startTime,endTime,status,triggerName
-    08585329112602833828909892130CU17,2022-11-17T14:33:45.2763872Z,,Running,When_a_new_response_is_submitted
+    08585236861638480597867166179CU104,2023-03-04T09:05:21.8066368Z,2023-03-04T09:05:22.5880202Z,Succeeded,When_an_email_is_flagged_(V4)
+    ```
+
+=== "Markdown"
+
+    ```md
+    # flow run get --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5 --flowName 5923cb07-ce1a-4a5c-ab81-257ce820109a --name 08586653536760200319026785874CU62
+
+    Date: 04/03/2023
+
+    ## 08586653536760200319026785874CU62 (/providers/Microsoft.ProcessSimple/environments/Default-d87a7535-dd31-4437-bfe1-95340acd55c5/flows/5923cb07-ce1a-4a5c-ab81-257ce820109a/runs/08586653536760200319026785874CU62)
+
+    Property | Value
+    ---------|-------
+    name | 08586653536760200319026785874CU62
+    startTime | 2023-03-04T09:05:21.8066368Z
+    endTime | 2023-03-04T09:05:22.5880202Z
+    status | Succeeded
+    triggerName | When\_an\_email\_is\_flagged\_(V4)
+    ```
+
+### `includeTriggerInformation` response
+
+When using the option `includeTriggerInformation`, the response for the json-output will differ.
```

Comment:
@MathijsVerbeeck I think in the current implementation the `triggerInformation` will also be present in `md` output right? Could you double check that and if so also include it in the docs?